### DISCUSSION TITLE
No Ipv4 in DFP-34X-2C2 but it gets IPV6 prefix

### DISCUSSION THREAD

**Discussion by jvalenciag** [Anime4000/RTL960x#355] [Help]
I am trying to replace my isp Huawei HG8145V5 ont with a DFP-34X-2C3 (firmware V1.0-220923) stick connected to a Mikrotik router switch.
After folllowing all instruction available in this repo I am not able to get an ipv4 address, however it gets an IPV6 prefix on the internet vlan. This router uses TR069 and vlans:

![Screenshot 2024-11-05 at 18 50 17](https://github.com/user-attachments/assets/5774cf1d-8798-4302-ac43-85c010c9d3f7)
Which I am able to see when running `omcicli mib get 84`:

```
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
VlanTagFilterData
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
=================================
EntityID: 0x01
FilterTbl[0]: PRI 0,CFI 0, VID 3000
FwdOp:  0x04
NumOfEntries: 1
=================================
=================================
EntityID: 0x02
FilterTbl[0]: PRI 0,CFI 0, VID 3100
FwdOp:  0x04
NumOfEntries: 1
=================================
=================================
EntityID: 0x03
FilterTbl[0]: PRI 0,CFI 0, VID 3200
FwdOp:  0x04
NumOfEntries: 1
=================================
=================================
EntityID: 0x04
FilterTbl[0]: PRI 0,CFI 0, VID 3300
FwdOp:  0x04
NumOfEntries: 1
=================================
```
But `omcicli mib get 171` returns nothing:
```
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
ExtVlanTagOperCfgData
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```

I am suspecting it is an issue with TR069 configuration. I know there is a TR069 config section in the stick under `http://192.168.1.1/tr069config.asp` but I don't know how to configure it.

**Comment by Anime4000** [comment]
I guess your DHCP request extension is via tr069?

perhaps try change fwdop to 0x02

**Comment by jxlarrea** [comment]
Were you able to solve this? I am experiencing the same IPv6 issue with a HiSense LTE3415 (RTL9601CI). Trying to replace a Huawei HG8145V5. IPv4 works but not IPv6. I get a /64 prefix but the router gets not IPv6 address and there is no IPv6 connectivity whatsoever.

**Reply by jvalenciag** [reply]
I was not able, but the issue I am having is that I am not able to get an ipv4 address, ipv6 I am getting an address.
